src/resources.cpp

```diff
@@ -109,7 +113,7 @@ DirectoryResourceProvider::DirectoryResourceProvider(string basepath)
 
 P<ResourceStream> DirectoryResourceProvider::getResourceStream(string filename)
 {
-    P<FileResourceStream> stream = new FileResourceStream(basepath + filename);
+    P<FileResourceStream> stream = new FileResourceStream(basepath + "/" + filename);
```

Comment:
It might be nice to strip off the trailing slash of the basepath in the constructor, that makes it work for both a trailing slash and not. Making the API less error prone.

Reply:
Ok seems legit :)